src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  margin: auto;
+  font-style: Roboto;
```

Comment:
The `font-style` property is incorrectly used here. If you intend to set the font family, use `font-family` instead of `font-style`. Also, ensure that 'Roboto' is correctly loaded in your project.